Sources/Resource.swift

```diff
@@ -12,32 +12,37 @@ import Foundation
 /// Protocol for resources inside Contentful
 protocol Resource: Decodable {
     /// System fields
-    var sys: [String : Any] { get }
+    var sys: [String: Any] { get }
     /// Unique identifier
     var identifier: String { get }
     /// Resource type
     var type: String { get }
 }
 
 protocol LocalizedResource {
-    var fields: [String:Any] { get }
+    var fields: [String: Any] { get }
 
     var locale: String { get set }
-    var localizedFields: [String:[String:Any]] { get }
+    var localizedFields: [String: [String: Any]] { get }
     var defaultLocale: String { get }
 }
 
-func +<K: Hashable, V> (left: Dictionary<K, V>, right: Dictionary<K, V>) -> Dictionary<K, V> {
+@discardableResult func +=<K: Hashable, V> (left: [K: V], right: [K: V]) -> [K: V] {
     var result = left
     right.forEach { (k, v) in result[k] = v }
     return result
 }
 
-func fields(_ localizedFields: [String:[String:Any]], forLocale locale: String, defaultLocale: String) -> [String:Any] {
+func +<K: Hashable, V> (left: [K: V], right: [K: V]) -> [K: V] {
+    return left += right
```

Comment:
I am not sure if I understand the noauces right, but I'd understand `mapA  = mapB + mapC` differently to `mapA = mapB += mapC`. 

For the first equation, I would think all three maps are different, and `mapA` contains elements from `mapB` and `mapC`. 

For the second, I would expect `mapA` and `mapB` to contain the exactly same elements after evaluation: all from `mapB` and `mapC` together.

It might be that the language has different semantics though, if so, please disregard ... ;)